crates/l2/sequencer/errors.rs

```diff
@@ -125,14 +123,12 @@ pub enum ProofSenderError {
     CalldataEncodeError(#[from] CalldataEncodeError),
     #[error("{0} proof is not present")]
     ProofNotPresent(ProverType),
-    #[error("Unexpected Error: {0}")]
-    InternalError(String),
+    #[error("Unknown Error: {0}")]
+    UnknownError(String),
     #[error("Failed to parse OnChainProposer response: {0}")]
     FailedToParseOnChainProposerResponse(String),
-    // TODO: Avoid propagating GenServerErrors outside GenServer modules
-    // See https://github.com/lambdaclass/ethrex/issues/3376
-    #[error("Spawned GenServer Error")]
-    GenServerError(GenServerError),
+    #[error("Unknown Error: {0}")]
```

Comment:
Shouldn't this be
```suggestion
    #[error("Internal Error: {0}")]
```
?